README.md

```diff
@@ -1,18 +1,36 @@
 [![Regression Tests](https://github.com/NHSDigital/electronic-prescription-service-api-regression-tests/actions/workflows/regression_tests.yml/badge.svg?branch=main)](https://github.com/NHSDigital/electronic-prescription-service-api-regression-tests/actions/workflows/regression_tests.yml)
 
 # Regression Tests
-These tests will automate regression testing of the Electronic Prescription Service API
-<br /> Any test failures will result in a failed build being reported (When run in CI)
+These tests will automate End-to-End regression testing of the following products:
+* [EPS-FHIR](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir)
+* [Prescriptions for Patients (PfP)](https://digital.nhs.uk/developer/api-catalogue/prescriptions-for-patients)
 
 ## General usage
 These tests are run automatically during deployment and shouldn't need to be touched unless performing debugging or
-adding/removing/changing the test cases
+adding/removing/changing test cases <br />
+If there are any test failures, this will report a failed build
+## Setup
 
-## Preparing your development environment
+### Environment Variables
+It is necessary to set some Environment variables in order to run any tests in your local environment. The tests will look for environment variables in the following order:
+(For security, the values will not be displayed here)
+1. `.env` file
+2. OS environment variable
+
+The following environment variables need to be set for the correct environment you wish to test against:
+* CLIENT_ID
+* CLIENT_SECRET
+* PRIVATE_KEY
+* CERTIFICATE
+
+To make this easier. a `template.env` file is located on the root. Fill in the values and rename this to `.env`
+Any file that beings with `.env` is automatically ignored by Git
+
```

Comment:
Please correct the typo in "beings":
Any file that begins with `.env` is automatically ignored by Git.